client/src/components/AuthForm/SignUpForm/SignUpForm.js

```diff
@@ -39,15 +39,18 @@ const SignUpForm = ({
     return result.length < 4 || signUpForm.password.length < 6;
   };
 
-  const handleSubmit = async (e) => {
+  const handleSubmit = (e) => {
     e.preventDefault();
-    try {
-      const result = await register(signUpForm);
-      if (alert.length > 0) {
-        console.log(alert[0].message);
-      }
-    } catch (err) {
-      console.log(err);
+    if (!validator.isEmail(signUpForm.email)) {
```

Comment:
use destructuring in the import statement, no need to import the entire validator package for just one method

import {isEmail} from 'validator'